article/sections/_2_graph_importance.tex

```diff
@@ -73,18 +69,9 @@ \section{Perils of disregard}
    &\quad + \beta_{\textrm{cost\_per\_distance\_sr3+}} \times \textrm{Cost\_per\_Distance}_{\textrm{sr3+}} + \beta_{\textrm{autos}}  \times \textrm{Number\_of\_Autos} \\
    &\quad + \beta_{\textrm{cross\_bay}} \times \textrm{Cross\_Bay} + \beta_{\textrm{hh\_size}} \times \textrm{Household\_Size} \\
    &\quad + \beta_{\textrm{n\_kids\_hh}} \times \textrm{Number\_of\_kids} \\
-   \textrm{Utility} \left(\textrm{Walk-Transit-Walk}\right) &= ASC_{\textrm{WTW}} + \beta_{\textrm{travel\_time\_transit}} \times \textrm{Travel\_Time} \\
-   &\quad + \beta_{\textrm{travel\_cost}} \times \textrm{Travel\_Cost} \\
-   \textrm{Utility} \left(\textrm{Drive-Transit-Walk}\right) &= ASC_{\textrm{DTW}} + \beta_{\textrm{travel\_time\_transit}} \times \textrm{Travel\_Time} \\
-   &\quad + \beta_{\textrm{travel\_cost}} \times \textrm{Travel\_Cost} \\
-   \textrm{Utility} \left(\textrm{Walk-Tranist-Drive}\right) &= ASC_{\textrm{WTD}} + \beta_{\textrm{travel\_time\_transit}} \times \textrm{Travel\_Time} \\
-   &\quad + \beta_{\textrm{travel\_cost}} \times \textrm{Travel\_Cost} \\
-   \textrm{Utility} \left(\textrm{Bike}\right) &= ASC_{\textrm{Bike}} + \beta_{\textrm{travel\_distance\_bike}} \times \textrm{Travel\_Distance} \\
-   \textrm{Utility} \left(\textrm{Walk}\right) &= ASC_{\textrm{Walk}} + \beta_{\textrm{travel\_distance\_walk}} \times \textrm{Travel\_Distance} \\
    \end{aligned}
 \end{equation*}
 
-
 For our purposes, we assume that there are no latent variables.
```

Comment:
I updated your suggestion to drop the citation @hassan-obeid.
```suggestion
Note that, since we consider this model to be the true outcome model, there are no latent variables we need to account for.```